### DISCUSSION TITLE
Disabled pinning for private actions

### DISCUSSION THREAD

**Discussion by NeilJed** [renovatebot/renovate#27109] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

We're currently using the `helpers:pinGitHubActionDigests` to follow the GitHub best practices but it's causing a lot of extra PRs for our own internal actions. Most devlopers pin to a major version and we just update under that.

Is there a way to configure the pinning to exlude any actions that start with our github org name or specific action repositories? Our ultimate goal is:

- Pin official/third party github actions
- Don't pin private actions/re-usable workflows within our own org.
- Pin our private actions/workflows to either a major or minor semantic version. i.e workflows can pin @v4 or @v4.2. Renovate should open a PR to bump 4.2->4.3 but not 4 as that would get the 4.3 update anyway.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Do your private actions follow naming conventions such as from one or more orgs?

**Reply by NeilJed** [reply]
@rarkins Yes, all of them start with our org name so `myorg/action-name@v4`
